recipes/starshot_card_block_type/config/field.field.block_content.card.field_card_link.yml

```diff
@@ -0,0 +1,22 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - block_content.type.card
+    - field.storage.block_content.field_card_link
```

Comment:
here "field_card_link" appears to have the intention of offering a call to action. This is a very generic thing and I would prefer to see a "field_ctalink" or `field_cta_link` or something like that can be reused - and hence if, for example, it is themed it is theme consistently. (Bearing in mind I don't yet know how well recipes will overlap with shared fields.)

Reply:
No objection to renaming the field. The machine names of these fields won't matter much to end users, and we can rename them whenever we want since there is no update path.

Reply:
Personally I go with a generic 'link' if singular and 'links' if plural for maximum re-use and minimum number of DB tables.

Reply:
If this was intended for reuse, I think we'd move it to a different, more generic recipe. IMHO that can be done later; this field was introduced for a specialized block type, and for now we can keep it there.